common/src/main/java/com/wynntils/models/lootrun/type/TaskLocation.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.lootrun.type;
+
+import com.google.common.collect.ComparisonChain;
+import com.wynntils.utils.mc.type.Location;
+
+public record TaskLocation(Location location, LootrunTaskType taskType) implements Comparable<TaskLocation> {
+    @Override
+    public int compareTo(TaskLocation taskLocation) {
+        return ComparisonChain.start()
+                .compare(location, taskLocation.location)
```

Comment:
You didn't change Location? Maybe it already was Comparable?

Reply:
It was.